### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][ColourContrast][MAS1.4.3][IntelliJ][Wipro]: Colour contrast ratio is below 4.5:1 for selected plugins in plugins dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1372]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box.
3. Plugins dialog will open.
4. Navigate to " plugins"  and observe colour contrast ratio.

**Expected Result:**
Colour contrast ratio should be greater than or equal to 4.5:1 for selected plugins .

**Actual Result:**
Colour contrast ratio is less than 4.4:1 for selected plugins . 

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)
**CCA  version**: 2.5.0.0

![cca for plugins in plugin window](https://user-images.githubusercontent.com/22956130/35680576-a344f126-070f-11e8-87cc-88411ff15524.png)

**Comment by Eskibear** [comment]
It's IntelliJ IDEA's own dialog, out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749817
